src/features/click-to-play.js

```diff
@@ -1600,15 +1587,15 @@ async function replaceClickToLoadElements (config, targetElement) {
 /**
  * Send a message to the background to unblock requests for the given entity for
  * the page.
- * @param {enableSocialTrackerRequest} message
- * @see {@event ddg-ctp-enableSocialTracker-complete} for the response handler.
+ * @param {unblockClickToLoadContentRequest} message
+ * @see {@event ddg-ctp-unblockClickToLoadContent-complete} for the response handler.
  */
-function enableSocialTracker (message) {
-    sendMessage('enableSocialTracker', message)
+function unblockClickToLoadContent (message) {
+    sendMessage('unblockClickToLoadContent', message)
```

Comment:
Since we're now in the realm of tidying up the messages, I have taken the opportunity to rename "enableSocialTracker" to "unblockClickToLoadContent", since that name was pretty confusing. The other names weren't too bad though, so I left them alone.